libvirt/utils_libvirt_test.go

```diff
@@ -32,3 +34,81 @@ func TestGetHostXMLDesc(t *testing.T) {
 		t.Errorf("expected name %s, got %s", name, dd.Name)
 	}
 }
+
+func connect(t *testing.T) *libvirt.Connect {
+	conn, err := libvirt.NewConnect(os.Getenv("LIBVIRT_DEFAULT_URI"))
+	if err != nil {
+		t.Fatalf("Cannot connect")
```

Comment:
This should include the actual error as well.

Reply:
Will fix